mods/stairs/init.lua

```diff
@@ -855,3 +855,23 @@ stairs.register_stair_and_slab(
 	default.node_sound_snow_defaults(),
 	true
 )
+
+stairs.register_stair_and_slab(
+	"glass",
+	"default:glass",
+	{cracky = 3, oddly_breakable_by_hand = 3},
+	{"default_glass.png", "default_glass.png", "stairs_glass_slab.png"},
+	"Glass Stair",
+	"Glass Slab",
+	default.node_sound_glass_defaults()
+)
```

Comment:
Missing bool for WA.

Reply:
https://github.com/minetest/minetest_game/pull/2226#issuecomment-426112817
So 'false'.

Reply:
Well, that was silly of me, duh
fixed for both